packages/engine-test-utils/src/__tests__/dendron-cli/commands/seedCLICommand.spec.ts

```diff
@@ -45,7 +45,7 @@ async function createSeed({ engine }: { engine: DEngineClient }) {
   return { registryFile, seedDict, seedId };
 }
 
-describe("remove", () => {
+describe.skip("remove", () => {
   const cmd = SeedCommands.REMOVE;
```

Comment:
why are we skipping these tests? if its to diagnose later, can we leave a comment? also, can we add a conditional to only skip on windows? 
```
let runTest = describe
if (os === windows) {
runTest = describe.skip
}
```

apply above comment to all instances were we skip tests